app-connector-client/test/test_integration.js

```diff
@@ -1,14 +1,35 @@
 

```

Comment:
Please treat this here as the one test suite which is nice to read and understand, no reasoning about historical code, please. The current structure is not good, please re-design.

Reply:
Should be more readable now.

Reply:
much better now, thank you